homeassistant/components/tankerkoenig/base_sensor.py

```diff
@@ -0,0 +1,88 @@
+"""Base classe for tankerkoenig sensor integration."""
+import logging
+
+from homeassistant.helpers.entity import Entity
+from homeassistant.const import (
+    ATTR_ATTRIBUTION,
+    ATTR_LATITUDE,
+    ATTR_LONGITUDE,
+    STATE_CLOSED,
+    STATE_OPEN,
+)
+from .const import NAME
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_ADDRESS = "address"
+ATTR_BRAND = "brand"
+ATTR_FUEL = "fuel_type"
+ATTR_IS_OPEN = "state"
+ATTR_STATION_NAME = "station_name"
+ATTRIBUTION = "Data provided by https://creativecommons.tankerkoenig.de"
+
+ICON = "mdi:fuel"
```

Comment:
I don't think you need a global for this (I could be wrong), just return it in the property.

Reply:
That is a choice, nothing wrong with this method.

Reply:
I copied this part from the Prezzibenzina integration, and it is also doing it this way. In general, I like having the constants as globals and grouped, to make it easier to find and change them if needed. If this is blocking the merge, I would change that, but otherwise, I'd prefer to keep it as is.